### PR TITLE
Add new example to match using comparison operators

### PR DESCRIPTION
**Pull request by Marqitos**
Validation of operation of the example using Comparison Operators
https://onlinephp.io/c/a6825

Related to Spanish translation:
https://github.com/php/doc-es/pull/86

### CODE REVIEW

File: language/control-structures/match.xml [hunk 1]
```diff
@@ -54,6 +54,33 @@ string(19) "This food is a cake"
...
+
+ $output = match (true) {
+     $age < 2 => "You're a baby",
+     $age < 13 => "You're a child",
+     $age <= 19 => "You're a teenager",
+     $age > 19 => "You're a young adult",
+     $age >= 40 => "You're an old adult"
+ };
+
+ var_dump($output);
```
Comment (Girgias) [inline language/control-structures/match.xml:72]: The indent is weird here.

Also we try to not use personalization in the docs (i.e. using "you") so could you change the strings to just be something like `"Baby"`, `"Child"`, etc.

**Review by Girgias — COMMENTED** [review]
Makes sense overall, thank you!

File: language/control-structures/match.xml [hunk 2]
```diff
@@ -54,6 +54,33 @@ string(19) "This food is a cake"
...
+
+var_dump($output);
+?>
+]]>
+  </programlisting>
+  &example.outputs;
+   <screen>
+<![CDATA[
+string(17) "You're a teenager"
+]]>
```
Comment (Girgias) [inline language/control-structures/match.xml:80]: Result needs fixing, but otherwise LGTM

Comment (Marqitos) [reply language/control-structures/match.xml]: Corrected, sorry for the work I have given you.

**Pull request merged**